Source/Core/DolphinWX/DolphinWX.vcxproj

```diff
@@ -13,12 +13,12 @@
   <PropertyGroup Label="Globals">
     <ProjectGuid>{47411FDB-1BF2-48D0-AB4E-C7C41160F898}</ProjectGuid>
     <ProjectName>Dolphin</ProjectName>
-    <WindowsTargetPlatformVersion>10.0.15063.0</WindowsTargetPlatformVersion>
+    <WindowsTargetPlatformVersion>10.0.16299.0</WindowsTargetPlatformVersion>
```

Comment:
If the visual studio files need updating, split them off into a separate PR.

Reply:
That was my mistake, i didn't mean to commit it, i've reverted it (though it is still listed in the changed files).